### ISSUE TITLE
用自己的训练预料loss降不下来

### ISSUE THREAD

**Issue by shizhediao** [warmheartli/ChatBotCourse#16]
我把训练预料换成了自己准备的，大概10-40w条对话
但是跑完10000个step之后，发现loss在4-5之间。。。
有没有什么办法
谢谢

**Comment by renhui2018** [comment]
我加了2W 条对话，loss 也处于4-5 之间，learning_rate已经为0 了

**Comment by Yuzeyang** [comment]
同样有这样的问题，loss降不下来，有解决的么？ @warmheartli

**Comment by alige32** [comment]
增大LSTM神经元的size,提高词嵌入的维度。另外内存允许的话适当增加batch_size
